dev/Dockerfile

```diff
@@ -10,7 +11,7 @@ RUN apt-get update && \
     pip3 install --upgrade pip
 
 # install ansible and its test tool
-RUN pip3 install ansible pytest-testinfra
+RUN pip3 install ansible==$ANSIBLE_VERSION pytest-testinfra
```

Comment:
There are differences in versioning between the `ansible` and `ansible-core` packages - we should install `ansible-core` here, which keeps with the v2.x.y theme set in our GitHub Actions flow.